src/mitol/common/__init__.py

```diff
@@ -1,5 +1,5 @@
 """ mitol.common """
 default_app_config = "mitol.common.apps.CommonApp"
 
-__version__ = "2.5.2"
+__version__ = "2.5.3"
```

Comment:
Shouldn't bump this, we have a script that does this and git-tags.

Reply:
Is this also true for `CHANGELOG.md` (should I leave that file blank under the new app?)